### DISCUSSION TITLE
When I package my project for Qfield, for some eldritch reason It leaves out all features on a specific layer.

### DISCUSSION THREAD

**Discussion by PandarianPwnerfluken** [opengisch/QField#4283] [Q&A]
Expanding on it.

Said layer contains around 200 point features, I used this project for months now without any such problems, and suddenly it decided to do this. Project also contains half a dozen tables which are packaged without issue, all of their records are intact. The only "new" thing I've done is move around a few points to better reflect reality on the field earlier this week, but I noticed no issue until I tried to package the layer with the QFieldSync pluggin. I cannot access the project rn, so the amount of info i can give is limited somewhat, sorry.

**Comment by ThomasDV1984** [comment]
Is it possible that the button only copy features in the area of interest is checked where you do the layer settings for qfield?

**Reply by PandarianPwnerfluken** [reply]
Sadly not, I checked and it was turned off. As far as I know there hasn't been any change in settings since the last time I packed the project, It's possible someone fiddled with the settings without my knowledge, but I haven't noticed any change. The layer was in offline editing mode, I tried packaging it in different directories, different versions of the project. I'd also do with a workaround if all else fails.
